### DISCUSSION TITLE
Nasıl fazladan İbax puanı kazanırım ?

### DISCUSSION THREAD

**Discussion by arfdgn** [IBAX-io/go-ibax#821] [Soru Cevap - Türkçe]
Çekilişe katılarak rakiplerinize fark atabilirsiniz.

**Comment by Yusufknk** [comment]
Çekiliş ve görevlere aynı zamanda githubta projeye destek olacak geliştirme fikir ve önerilerinizi paylaşarak da kazanabilirsiniz.

**Reply by Brown2222** [reply]
Follow Retwit posts, new ones are coming all the time
